sanic/request.py

```diff
@@ -104,6 +104,7 @@ class Request:
         "parsed_json",
         "parsed_forwarded",
         "raw_url",
+        "responded",
```

Comment:
Looks like this variable is unnecessary, as you can simply check `self.response` for the same thing.